cmake/ConkyPlatformChecks.cmake

```diff
@@ -97,6 +97,7 @@ endif(CMAKE_SYSTEM_NAME MATCHES "SunOS")
 
 if(CMAKE_SYSTEM_NAME MATCHES "NetBSD")
   set(OS_NETBSD true)
+  set(conky_libs ${conky_libs} -lkvm -lintl -lossaudio)
```

Comment:
- `-lintl` not needed.
- `-lossaudio` should be handled  on line 159 in this file presumably by adding `OR OS_NETBSD` to `elseif(OS_OPENBSD)`